PelumiAlesh/Banka
# #165954869 Consume staff view all accounts

#### What does this PR do?
Integrate the view all accounts endpoint to Staff UI
#### Description of Task to be completed?
All staff should be able to view all the accounts and their details
#### How should this be manually tested?
Visit pelumialesh.github.io/Banka/UI and sign in with staff credentials, and check the dashboard 
#### Any background context you want to provide?
Null
#### What are the relevant pivotal tracker stories?
#165954869 
#### Screenshots (if appropriate)
Null
#### Questions:
Null
